nix/hetzner-bootstrap.nix

```diff
@@ -19,6 +19,7 @@ let
       mkdir -p "$out/bin"
       ln -s "${nix}"/bin/* "$out/bin/"
       ln -s "${stdenv.shell}" "$out/bin/sh"
+      ln -s "${pkgsNative.cacert}" "$out/etc-certs"
```

Comment:
I believe the use of `exportReferencesGraph` tells mkDerivation to not bundle in referenced packages, but only a specific list. I referenced the package, found it wasn't on the hetzner box, then added it to this derivation, and found that to work.